.eslintrc.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
```

Comment:
I hope it's not too late for that, but there is an [ESLint config](https://github.com/grafana/eslint-config-grafana), that is used by Grafana itself. Maybe we should look into that?

Reply:
good idea!